doc/script_commands.txt

```diff
@@ -2935,6 +2935,13 @@ Merge all stackable items that separated by GUID flags
 (either by flag 4 item_flag.txt or GUID  in item_group).
 If no item ID/name given, all possible items in player's inventory will be merged.
 
+---------------------------------------
+
+*gettradability(<equipment slot>{,<char id>});
```

Comment:
Hi, don't be offended, just suggestion because will use the command many times and the name is very bad for not native english speakers. Because if I wish to check if equip is tradable I starting to think about totally different name like

`equip_istradable(EQI_SLOT, char_id)`
or
`equip_cantrade(EQI_SLOT,char_id)`
or
`equip_tradable(EQI_SLOT,char_id)`

Reply:
I'm following the format of old commands. Thanks for the suggestion ♥